ProteoformSuiteInternal/DeltaMassPeak.cs

```diff
@@ -29,21 +29,23 @@ public List<ProteoformRelation> grouped_relations
         public List<Modification> possiblePeakAssignments { get; set; }
         public string possiblePeakAssignments_string
         {
-            get { return String.Join("; ", possiblePeakAssignments.Select(m => m.description).ToArray()); }
+            get { return String.Join(", ", possiblePeakAssignments.Select(m => m.description).ToArray()); }
```

Comment:
I think this should be a semicolon. Don't the descriptions contain commas? The semicolon would allow us to split fields with commas in them.